1_fetch/src/get_nwis_data.R

```diff
@@ -10,13 +10,14 @@ combine_csvs <- function(input_dir){
     these_data <- read_csv(download_file, col_types = 'ccTdcc')
     data_out <- rbind(data_out, these_data)
   }
-  return(data_out)
+  write.csv(data_out, outfile)
```

Comment:
`readr::write_csv` is a bit nicer than `write.csv` - for example, it uses Posix line endings by default and avoids writing row numbers by default.